internal/signerverifier/ssh/ssh.go

```diff
@@ -179,3 +191,20 @@ func parseSSH2Key(data string) (ssh.PublicKey, error) {
 	body := strings.Join(lines[i:], "")
 	return parseSSH2Body(body)
 }
+
+func calculateKeyID(k *sv.SSLibKey) (string, error) {
```

Comment:
copied from the tuf package; we can maybe remove key APIs there and move them all to signerverifier? especially if we make MetadataKey() a thing here.